packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountItem.tsx

```diff
@@ -129,6 +125,17 @@ export const AccountItem = forwardRef(
             }
         };
 
+        const fiatRates = useSelector(selectCurrentFiatRates);
+        const coinDefinitions = useSelector(state => selectCoinDefinitions(state, account.symbol));
+        const tokensWithRates = enhanceTokensWithRates(
+            account.tokens,
+            localCurrency,
+            symbol,
+            fiatRates,
+        );
+        const sortedTokens = tokensWithRates.sort(sortTokensWithRates);
+        const tokens = getTokens(sortedTokens, account.symbol, coinDefinitions);
+
```

Comment:
I would create a wrapper component for this as it is going to be used on at least 2 more places